### ISSUE TITLE
trying to solve the create button working

### ISSUE THREAD

**Issue by flyjwayur** [Rookies-RGSOC2016/Hoodie-electron-demo#16]

**Comment by flyjwayur** [comment]
Plus trying to put 'sidebar.html' inside of 'index.html' with the below code but, it's not working 
-> getting cross origin request error in the console, it's not displaying the sidebar 
$('#left-sidebar').load('sidebar.html');

**Comment by flyjwayur** [comment]
``` js
var appname = location.hash.substr(1);
console.log(appname);

//redirect the exsiting apps to the detail_app_page.html
//It redirect but, not displaying appname in URL
console.log('redirect-appname'+appname);

function redirect(){
    console.log('redirect-appname'+appname);
    var url = "detail_app_page.html#"+appsname;
    $(location).attr('href',url);
    $('#name-app').html(appname);
}
```

I thought appname is a global variable, but in console, it's said that "Uncaught ReferenceError: appsname is not defined"(when I use console.log inside of function redirect()). Maybe I'm thinking something wrong.

**Comment by gr2m** [comment]
I can have a loook at it.

Also this line is wrong:

``` js
$(location).attr('href',url);
```

`location` is not a DOM element, so you don’t need to use jQuery to set its `href` attribute. Instead, do this directly:

``` js
location.href = url
```

**Comment by flyjwayur** [comment]
closed #17
